src/results/proforma.jl

```diff
@@ -250,7 +255,9 @@ function proforma_results(p::REoptInputs, d::Dict)
     total_cash_incentives = m.total_pbi * (1 - tax_rate_fraction)
     free_cashflow_without_year_zero = m.total_depreciation * tax_rate_fraction + total_cash_incentives + operating_expenses_after_tax
     free_cashflow_without_year_zero[1] += m.federal_itc
-    r["initial_capital_costs_after_incentives_without_macrs"] = d["Financial"]["initial_capital_costs"] - m.total_ibi_and_cbi - m.federal_itc
+    battery_replacement_net_present_cost = -1*battery_replacement_cost * (1 - tax_rate_fraction) / (1 + p.s.financial.offtaker_discount_rate_fraction) ^ battery_replacement_year  # battery_replacement_cost is negative, from above
```

Comment:
@Bill-Becker should the discount rate used here depend on whether the analysis is third party, similar to the tax rate?

Reply:
Updated to depend on third_party